grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -914,7 +911,7 @@ class ReportService implements ApplicationContextAware {
                     AND (invoice.invoice_type_id != :prepaymentInvoiceId OR invoice.invoice_type_id IS NULL)
                     AND (invoice_item.inverse IS NULL OR invoice_item.inverse = FALSE)
                     ${additionalFilter}
-                GROUP BY o.id, order_item.id, shipment_item.id, invoice_item.id
+                GROUP BY o.id, order_item.id, shipment_item.id
```

Comment:
why were we grouping by invoice item before? It seems like doing that would only ever result in one line per grouping and so make the SUM in line 902 pointless. What you've done here looks correct. I suppose the only reason it was working before was that an order item only ever had one invoice item so it just happened to work?

Reply:
Yeah I am not sure. IMHO it should not have been done since the beginning, and I am not sure why it was done. I think the regular invoices (not prepaid) were broken because of this before, but this bug was hard to spot.